components/Buttons/src/MDCButton.m

```diff
@@ -252,10 +262,13 @@ - (void)layoutSubviews {
 
```

Comment:
We will never get in to this if conditional if using the rippleView as inkView.usesLegacyInkRipple will not be YES. Therefore setting the rippleView frame isn't needed here.

Reply:
Removed